metadata-editor/app/lib/UsageRightsMetadataMapper.scala

```diff
@@ -0,0 +1,19 @@
+package lib
+
+import com.gu.mediaservice.model._
+
+object UsageRightsMetadataMapper {
+
+  def usageRightsToMetadata(usageRights: UsageRights): Option[ImageMetadata] = {
+    val toImageMetadata: PartialFunction[UsageRights, ImageMetadata] = (ur: UsageRights) => ur match {
+      case u: StaffPhotographer        => ImageMetadata(byline = Some(u.photographer), credit = Some(u.publication))
+      case u: ContractPhotographer     => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
+      case u: CommissionedPhotographer => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
```

Comment:
Could we use the [Photographer trait](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala#L89) here?